### DISCUSSION TITLE
Update 20210131

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#153] [Fixed Issues]
After update to 20210131 (on RPi Zero W and 3B+) the System page cannot be displayed, get this message:

Errors: Unexpected string in JSON at position 925
{ "cpuload" : "0.35 0.11 0.03" , "cputemp" : 52.6 , "startup" : "6s (kernel) + 19s (userspace) " , "throttled" : "0x80000" , "time" : "09:43:53 2021-01-31" , "timezone" : "Europe/London" , "uptime" : "1 week 4d 42m" , "uptimesince" : "2021-01-20 09:01" , "audioaplayname" : "merus-amp" , "audiooutput" : "MERUS Audio Amp" , "bluetooth" : false , "btdiscoverable" : false , "btformat" : false , "hostapd" : false , "hostname" : "RAConservatory" , "kernel" : "5.4.83-1-ARCH armv7l" , "lcd" : false , "lcdchar" : false , "lcdcharaddr" : "" , "lcdcharconf" : "" , "ntp" : "pool.ntp.org" , "reboot" : "" , "regdom" : "00" , "relays" : false , "rpi01" : false , "rpimodel" : "Raspberry Pi 3 Model B Plus Rev 1.3" , "soc" : "BCM2837B0" , "soccpu" : "Cortex-A53" , "socram" : "964MiB" , "socspeed" : "1400" , "soundprofile" : true , "version" : "1" , "versionui" : 20210131 , "wlan" : true , "soundprofileval" : "145655 60 1000 4000"▮"}

Rebooting device has no effect.

Chris

**Comment by rern** [comment]
Sorry. It's fixed. Try update again.

By the way, image files have been updated to the latest kernel and packages. Please try it and let me know whether it's still OK on your Zero.

**Reply by ceejayemm** [reply]
Thanks, that fixes the original problem but after applying 202101311 the RPi Browser option (RPi 3B+) option in Features is turned off and it wont let me turn it back on again.

I will try the new Pi Zero image later .

Chris
